lib/internal/fs/utils.js

```diff
@@ -345,6 +345,10 @@ function Stats(dev, mode, nlink, uid, gid, rdev, blksize,
 Object.setPrototypeOf(Stats.prototype, StatsBase.prototype);
 Object.setPrototypeOf(Stats, StatsBase);
 
+// HACK: Workaround for https://github.com/standard-things/esm/issues/821.
```

Comment:
nit: period becomes part of link
```suggestion
// HACK: Workaround for https://github.com/standard-things/esm/issues/821
```